Runtime/Plataforms/ServerLinuxBuildPlatform.cs

```diff
@@ -0,0 +1,48 @@
+using UnityEditor;
+using UnityEngine;
+
+namespace GameWorkstore.Automation
+{
+    [CreateAssetMenu(fileName = nameof(ServerLinuxBuildPlatform), menuName = "Automation/" + nameof(ServerLinuxBuildPlatform))]
+    public class ServerLinuxBuildPlatform : StandaloneBuildPlatform
+    {
+        public override void OnBuild()
+        {
+            //Version
+            if (!UnityEditorInternal.InternalEditorUtility.isHumanControllingUs)
+            {
+                Version(out var version, out _);
+                PlayerSettings.bundleVersion = version;
+            }
+
+            //SetScriptDefinitions
+            if (UseCustomScriptDefinitions)
+            {
+                PlayerSettings.SetScriptingDefineSymbols(
+                    UnityEditor.Build.NamedBuildTarget.Server,
+                    ScriptDefinitions.Definitions
+                );
+            }
+
+            //Backend
+            PlayerSettings.SetScriptingBackend(BuildTargetGroup.Standalone, ScriptingBackend);
+
+            //Options
+            var buildOptions = new BuildPlayerOptions
+            {
+                scenes = GetScenes(),
+                locationPathName = "Build/GameServerLinux/" + buildScript.GameName.ToLower().Replace(" ","_") + "_" + PlayerSettings.bundleVersion.Replace(".","_"),
```

Comment:
breaking changes: new name will cause issues to people that are current using this package. Make this change a boolean option on this platform instead.